package.json

```diff
@@ -326,6 +326,15 @@
             "react"
           ]
         },
+        "dash.docset.julia": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "default": [
+            "julia"
+          ]
+        },
```

Comment:
@sbromberger thanks for making the PR 😃 

is there any Julia language extension to install in VS Code? if yes, we should add that in README https://github.com/deerawan/vscode-dash#supported-docsets

Reply:
There is. Let me revise this PR.